ceph.spec.in

```diff
@@ -1130,7 +1130,7 @@ ${CMAKE} .. \
     -DCMAKE_INSTALL_MANDIR=%{_mandir} \
     -DCMAKE_INSTALL_DOCDIR=%{_docdir}/ceph \
     -DCMAKE_INSTALL_INCLUDEDIR=%{_includedir} \
-    -DWITH_MANPAGE=ON \
+    -DWITH_MANPAGE=%{python3_version} \
```

Comment:
-DWITH_MANPAGE ?

Reply:
^ yeah, that

Reply:
also seems like it maybe should be %{_python_buildid}, but I don't know

Reply:
`_python_buildid` is defined using `python3_pkgversion`, which is in turn used for naming python3 packages
```
$ rpm --eval '%{python3_pkgversion}'
34
```
while what i want here is a version number
```
$ rpm --eval '%{python3_version}'
3.4
```

Reply:
@badone fixed and repushed.